conf/development.ini

```diff
@@ -24,6 +24,13 @@ feature_flags_cookie_secret = "notasecret"
 # The secret string that's used to sign the OAuth2 state param.
 oauth2_state_secret = "notasecret"
 
+via_url = http://localhost:9080
+h_authority = lms.hypothes.is
+h_api_url_public = http://localhost:5000/api/
+h_api_url_private = http://localhost:5000/api/
+rpc_allowed_origins = http://localhost:5000
+sentry_environment = dev
```

Comment:
These aren't sensitive at all so should just have sensible dev defaults in the config file